themes/bootstrap5/templates/turnstile/challenge.phtml

```diff
@@ -0,0 +1,23 @@
+<?php
```

Comment:
I think we need a bootstrap3 version of this template. It won't be needed for very much longer, but until we fully remove the bootstrap3 themes, I think we'll be less confused if we keep things in sync.

Reply:
Definitely -- my bad.